curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e482c620eae02878e9d4c.md

```diff
@@ -0,0 +1,209 @@
+---
+id: 646e482c620eae02878e9d4c
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+For the second case, use the value of `mm-dd-yyyy-h-mm`. Inside that case, assign empty template literals to the constant named `currentDateParagraph` using the `textContent` property.
+
+# --hints--
+
+You need to use the `case` where the condition is `mm-dd-yyyy-h-mm` to assign empty template literal to the constant named `currentDateParagraph` using the `textContent` property.
+
+```js
+assert(code.match(/case\s*(['"])mm-dd-yyyy-h-mm\1\s*:\s*currentDateParagraph.textContent\s*=\s*``/g));
+```
+
+You need to use the `break` after the end of your code 
+
+```js
+assert(code.match(/break/g));
```

Comment:
This test will currently catch the existing `break` statement. It could be updated to ensure that we're catching the second `break` statement after the the empty template literals learners add in this step.